static/js/settings_user_groups.js

```diff
@@ -209,57 +233,23 @@ exports.populate_user_groups = function () {
             });
         }
 
-        function do_not_blur(except_class, event) {
-            // Event generated from or inside the typeahead.
-            if ($(event.relatedTarget).closest(".typeahead").length) {
-                return true;
-            }
-
-            const blur_exceptions = _.without(
-                [".pill-container", ".name", ".description", ".input", ".delete"],
-                except_class,
-            );
-            if ($(event.relatedTarget).closest("#user-groups #" + data.id).length) {
-                return blur_exceptions.some(
-                    (class_name) => $(event.relatedTarget).closest(class_name).length,
-                );
-            }
-            return false;
-        }
-
-        function auto_save(class_name, event) {
-            if (!exports.can_edit(data.id)) {
```

Comment:
Don't we need to still do this check?

Reply:
Since the auto_save function is disabled for users who cannot edit, I think we do not need to check this. I tested it was working fine, or am I missing something out?